synadm/api.py

```diff
@@ -147,6 +147,12 @@ def room_delete(self, room_id, new_room_user_id, room_name, message,
             data.update({"message": message})
         return self.query("post", f"v1/rooms/{room_id}/delete", data=data)
 
+    def room_make_admin(self, room_id, user_id):
+        """ Make a user into an admin of a room
+        """
+        data = {"user_id": user_id}
```

Comment:
ask for existence of user_id and only add the data dict if it's there, otherwise make data an empty dict